common/buildcraft/builders/container/ContainerQuarry.java

```diff
@@ -10,11 +10,9 @@
 import buildcraft.lib.gui.slot.SlotBase;
 
 public class ContainerQuarry extends ContainerBCTile<TileQuarry> {
-    private static final int PLAYER_INV_START = 50;
-
     public ContainerQuarry(EntityPlayer player, TileQuarry tile) {
         super(player, tile);
-        addFullPlayerInventory(PLAYER_INV_START);
+        addFullPlayerInventory(50);
```

Comment:
Don't make unnecessary renames or reformats please - it just increases the PR size and its not actually useful.